### ISSUE TITLE
Safari 8 gives: ReferenceError: Can't find variable: Intl

### ISSUE THREAD

**Issue by talkhot** [iam4x/isomorphic-flux-boilerplate#97]
When opening the lasted boilerplate in Safari 8 (also not working on mobile iOS). I get the following error on the console: ReferenceError: Can't find variable: Intl. I tried to find the cause but had no luck so far.

**Comment by igorprado** [comment]
AFAIK Safari needs Intl polyfill.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl#Browser_Compatibility

https://www.npmjs.com/package/intl

**Comment by talkhot** [comment]
Its true Safari needs a polyfill but it should be loaded by the boilerplate here.
https://github.com/iam4x/isomorphic-flux-boilerplate/blob/master/app/utils/intl-polyfill.js

**Comment by talkhot** [comment]
I found what causes the problem with loading the Intl polyfill in Safari, but don't know why.
The fix for issue #64 makes Safari (and mobile Safari) make's Safari give the "Can't find variable" error.
```javascript
if (process.env.BROWSER) window.ReactIntl = require('react-intl');
```
In https://github.com/iam4x/isomorphic-flux-boilerplate/blob/master/app/utils/intl-loader.js

A quick solution is the disable the line above...

**Comment by djbobbydrake** [comment]
FYI @talkhot disabling that line will subsequently cause the same error in Chrome.

**Comment by djbobbydrake** [comment]
This seems to do the trick:
```
diff --git a/app/utils/intl-loader.js b/app/utils/intl-loader.js
index ccfe597..4b98791 100644
--- a/app/utils/intl-loader.js
+++ b/app/utils/intl-loader.js
@@ -1,10 +1,5 @@
 import debug from 'debug';
 
-// We need to define `ReactIntl` on the global scope
-// in order to load specific locale data from `ReactIntl`
-// see: https://github.com/iam4x/isomorphic-flux-boilerplate/issues/64
-if (process.env.BROWSER) window.ReactIntl = require('react-intl');
-
 const loaders = {
 
   en(callback, force = false) {
@@ -21,6 +16,12 @@ const loaders = {
       });
     }
     else {
+      // We need to define `ReactIntl` on the global scope
+      // in order to load specific locale data from `ReactIntl`
+      // see: https://github.com/iam4x/isomorphic-flux-boilerplate/issues/64
+      if (process.env.BROWSER) {
+        window.ReactIntl = require('react-intl');
+      }
       require.ensure([
         'react-intl/dist/locale-data/en.js',
         'data/en'
```

**Comment by iam4x** [comment]
Fixed with 8d0bed75a272176bc2c1fa540d41c524f0c55bd2

It was race condition ;)
